### DISCUSSION TITLE
vocabsieve reader does not open properly

### DISCUSSION THREAD

**Discussion by ghost** [FreeLanguageTools/vocabsieve#175] [General]
I have used Vocabsieve for a few days on my win 10 64-machine. Most things work including Anki integration etc. However when I click "Reader" all I get is a window with no content except "Books". Nothing is clickable and there is no place to paste anything.

**Comment by 1over137** [comment]
Did you use "set book path"? You need to set it to a folder container your books then it shows them.

**Reply by ghost** [reply]
Yes the path I set was e:\ouino\LearningFrenchThroughStories\reader\ and this folder contains one txt file

**Reply by ghost** [reply]
I have still the same problem

**Reply by 1over137** [reply]
only epub is supported

**Reply by ghost** [reply]
Oh! Now it works when I converted the textfile to epub using Calibre. I notice that an hover-over / lookup words does not seem to be in function. Is this not implemented or is it dependant on some setting? Anyway, thank you so much for a fantasic program-

**Reply by 1over137** [reply]
there is no hover support in reading and never was. Maybe it will be added in the future
you're supposed to click, which will copy and show up in vocabsieve window

**Comment by ghost** [comment]
I see. Anyway now it works: Thank you so much.
Erik Forsberg
Ekvägen 45
15254 Södertälje
0707357974


Den tis 15 okt. 2024 kl 21:07 skrev 1over137 ***@***.***>:
